run_dir/static/js/base.js

```diff
@@ -184,61 +180,45 @@ marked.setOptions({
   // Searches for P[\d+] and replaces with a link to the project page
   s = s.replace(/([\W])(P[\d]{3,5})(?!\w)/g, '$1<a class="text-decoration-none" href="/project/$2">$2</a>');
 
-  // Searches for FlowCell IDs and replaces with a link (Most complicated regex ever)
-  // - $1 = Captures a non-word character (javascript can't do lookbehind)
-  // - $2 = Matches flowcell date - eg 150505
-  // - $3 = Matches optional flowcell chunk - eg. _D00450_0168
-  // - $4 = Matches remaining flowcell ID - eg. _AC6H3RANXX or _AC6H3RANXX-SDVLKCH
-  // - Not capture lookahead to make sure that were not followed by any more word characters
-  // Replaces with link to flowcell ID without internal chunk
-  // Example: 150505_D00450_0168_AC6H3RANXX links to 150505_AC6H3RANXX
+  // Searches for FlowCell IDs and replaces with a link
   s = s.replace(/([\W])(\d{6,8})(_(?:ST-)?\w{5,10}_\d{3,4})(_\w{8,12}(?:\-\w{3,8})?)(?!\w)/g, '$1<a class="text-decoration-none" href="/flowcells/$2$4">$2$3$4</a>');
 
-  // Searches for ONT FlowCell IDs and replaces with a link (modified from Illumina FCs)
-  // - $1 = Captures a non-word character (javascript can't do lookbehind)
-  // - $2 = Matches flowcell date - eg 20230621
-  // - $3 = Matches 4 digit chunk - eg. _1629
-  // - $4 = Matches remaining flowcell ID - eg. _MN19414 or _3G
-  // - $5 = Matches remaining flowcell ID - eg. _APT766 or _PAO31800
-  // - $6 = Matches remaining flowcell ID - eg. _494e1a8c
-  // - Not capture lookahead to make sure that were not followed by any more word characters
-  // Example: 20230621_1629_MN19414_APT766_494e1a8c or 20230815_1205_3G_PAO31800_201c040e
+  // Searches for ONT FlowCell IDs and replaces with a link
   s = s.replace(/([\W])(\d{8})(_\d{4})(_\w{2,8})(_\w{6,8})(_\w{8})(?!\w)/g, '$1<a class="text-decoration-none" href="/flowcells_ont/$2$3$4$5$6">$2$3$4$5$6</a>');
-  
+
   return s;
-}
-function create_user_tags(s){
+};
+
+const create_user_tags = (s) => {
   // Searches for @\w+ and replaces with a mail link
-  s = s.replace(/(@)([a-zA-Z0-9.-]+)/g, '<a class="text-decoration-none" href="mailto:$2@scilifelab.se">$1$2</a>');
-  return s;
-}
-function check_img_sources(obj){
+  return s.replace(/(@)([a-zA-Z0-9.-]+)/g, '<a class="text-decoration-none" href="mailto:$2@scilifelab.se">$1$2</a>');
+};
+
+const check_img_sources = (obj) => {
   // Sort out any missing images
   // pass some images, eg $('#running_note_preview_body img')
   // Has to be called AFTER the code has been inserted into the DOM
-  pathArray = window.location.href.split( '/' );
-  var missing_img_src = pathArray[0]+'//'+pathArray[2]+'/static/img/missing_image.png';
-  $(obj).on('error', function () {
-    if($(this).is('img') && $(this).attr('src') !== missing_img_src){
+  const pathArray = window.location.href.split('/');
+  const missing_img_src = `${pathArray[0]}//${pathArray[2]}/static/img/missing_image.png`;
+  $(obj).on('error', () => {
+    if ($(this).is('img') && $(this).attr('src') !== missing_img_src) {
```

Comment:
Will the `this` here work properly? I read that Arrow functions do not have their own `this` and uses the surrounding scope as its scope.

Reply:
Hmm you're probably right, I don't know how to test it though. Is this used for reviewing images in the running notes? I changed the code anyhow.

Reply:
> Is this used for reviewing images in the running notes?

Seems like it. Not sure if it is used that much anymore

Reply:
Hmm maybe we could remove it? 😛